tensorflow/cc/inference.cpp

```diff
@@ -800,21 +847,45 @@ int main(int argc, char *argv[]) {
     output_classes = uint(output.shape().dim_size(3));
     std::cout << "Output shape is " << output.shape() << " with " << output.shape().dims() << " dimensions and " << output_classes << " classes" << std::endl;
 
+
     // resize and merge to get output size depending on switch/case
     std::cout << "Make output array" << std::endl;
     auto *merged_output_classes = new float[image_height * image_width * output_classes];
+    float *dual_output_classes {};
```

Comment:
```suggestion
    float *dual_output_classes = nullptr;
```
Been meaning to say, for regular type initialization like this in non-templated code, this style reads better. The value-initialization style is more appropriate for classes and inside of templates (the latter when you don’t necessarily know the type of the value you’re initializing). Either one works, but I recall you like seeing `nullptr` in these situations   ;)

Reply:
Done